packages/mobile_app/android/app/build.gradle

```diff
@@ -41,16 +40,12 @@ android {
         // TODO: Specify your own unique Application ID (https://developer.android.com/studio/build/application-id.html).
         applicationId "app.myprophet"
         minSdkVersion 16
-        targetSdkVersion 29
+        targetSdkVersion 28
```

Comment:
Any reason why we are lowering target sdk version?

Reply:
the main(app) package was recreated by "flutter create" in order not to miss anything redundant.
this is default targetSdkVersion it sets,